Lesson-7/lesson-7-new/index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <link rel="stylesheet" href="style.css">
+    <section class="top">
+    <div class="container_heading">
+        <img id="logo_img" class="logo" src="LinkedSprite.svg#logo_img" alt="logo">
+        <p class="stays"><a href=""> Stays</a> </p>
+        <p class="attractions"> <a href="">Attractions</a> </p>
+        <img class="night_img" src="LinkedSprite.svg#moon" alt="night">
+        <img class="account" src="LinkedSprite.svg#account" alt="accountcircle">
+    </div>
+    <div class="container_middle">
+        <p class="heading">Discover stays <br>
+ to live, work or just relax</p>
+        <div class="label">
+            <label id="search" for="search" class="search">Your destination or hotel name</label>
+            <label id="date" for="date">Check-in — Check-out</label>
+        </div>
+        <div class="place_to_request">
+            <input type="search" name="search_bar" placeholder="New York">
+            <input class="calendar" type="date" name="calendar" placeholder="Check-in — Check-out">
+            <input type="text" name="text" placeholder="Adults — Children — Room">    
+            <input class="button" type="submit" value="Search" placeholder="Search">
+        </div>
+        <div class="phone_app">
+        <img class="icon_google" src="image/google-play-badge.png" alt="google">
+        <img class="icon_app_store" src="image/Download_on_the_App_Store_Badge_US-UK_RGB_blk_092917 1.png" alt="app">
+        </div>
+    </div>
+    </section>
+<section class="Wat_do_we_offer">
+    <h2 class="headline">What do we offer</h2>
+    <div class="all_containers">
+    <div class="container_headphones">
+<img src="LinkedSprite.svg#headphones" alt="headphones" class="headphones">
+<p class="text_headphones">Support 24/7</p> 
+</div>
+<div class="container_communicate">
+<img src="LinkedSprite.svg#communicate" alt="communicate" class="communicate">
+<p class="text_communicate">Communicate directly</p> 
+</div>
+<div class="container_gadgets">
+<img src="LinkedSprite.svg#gadgets" alt="gadgets"  class="gadgets">
+<p class="text_gadgets">Book online</p> 
+</div>
+<div class="container_like">
+<img src="LinkedSprite.svg#like" alt="like" class="like_img">
+<p class="text_like">Real guest reviews</p>
+</div>
+</div>
+</section>
+<section class="last">
+</div>
+<div class="heading_logo">
+    <img class="heading_logo_last" src="image/logo_vector.png" alt="heading_logo">
+</div>
```

Comment:
What is it?
![image](https://user-images.githubusercontent.com/20189313/129097083-c1f76043-6929-42f4-8991-35238e39247b.png)